src/Tester/UnitTestSiloHost.cs

```diff
@@ -25,24 +28,13 @@ namespace UnitTests.Tester
     [DeploymentItem("OrleansProviders.dll")]
     [DeploymentItem("TestInternalGrainInterfaces.dll")]
     [DeploymentItem("TestInternalGrains.dll")]
-    public class UnitTestSiloHost : TestingSiloHost
+    public abstract class UnitTestSiloHost
     {
-        public UnitTestSiloHost() // : base()
-        {
-        }
-
-        public UnitTestSiloHost(bool startFreshOrleans)
-            : base(startFreshOrleans)
-        {
-        }
+        protected static readonly Random random = new Random();
```

Comment:
yes, I could copy that class to this project (or link it, as it's an internal that I don't have access too). I thought about doing that. I wouldn't want to make that utility class public as it has nothing to do with Orleans.